s_vae/models/backbone/utils.py

```diff
@@ -1,10 +1,31 @@
-from torch import nn
+import torch
 
-
-class Reshape(nn.Module):
+class Reshape(torch.nn.Module):
     def __init__(self, shape):
         super().__init__()
         self.shape = shape
 
     def forward(self, x):
-        return x.view(-1, *self.shape)
\ No newline at end of file
+        return x.view(-1, *self.shape)
+
+
+def sample_hypersphere(n, dim =3, R = 1, seed = 0):
+    """ 
+    This function will sample from uniform distribution on the *dim*-dimensional unit sphere. 
+    The method used comes from  "Computer Generation of Distributions on
+    the m-sphere" By GARY ULRICH. 
+    n: number of points to sample
+    dim: the dimension of the sphere.
+    R: radius of the sphere
+    """
+    torch.seed(seed)
```

Comment:
Seed is set for `torch` by `pytorch lightning`, so this should either be deleted or:
 
```suggestion
def sample_hypersphere(n, dim=3, R=1, seed=None):
    if seed is not None:
        torch.seed(seed)
```